netci.groovy

```diff
@@ -4,6 +4,8 @@
 import jobs.generation.Utilities;
 def project = GithubProject
 
+buildTimeLimit = 120
```

Comment:
@lgolding :+1: drop by my office when you want to merge this on Monday.  We can sit down and look at the results together.  It's easier to walk through the generator job in person.